core/src/custom_values/custom_value_address.rs

```diff
@@ -33,28 +33,13 @@ impl CustomValueAddress for str {
     }
 }
 
-/// Used to signal whether a [`CustomValueAddress`] can be decoded.
-pub struct Yes;
-
 /// A static address to a custom value.
 #[derive(Derivative)]
-#[derivative(
-    Clone(bound = ""),
-    Debug(bound = ""),
-    Eq(bound = ""),
-    Ord(bound = ""),
-    PartialEq(bound = "")
-)]
+#[derivative(Clone(bound = ""), Debug(bound = ""))]
```

Comment:
IIRC Niklas picked up on this already but I thought the bounds were all back, so: any reason for no more Eq/Ord/PartialEq here? :)